tests/lib/rules/attributes-order.js

```diff
@@ -276,6 +276,94 @@ tester.run('attributes-order', rule, {
             'GLOBAL'
           ]
         }]
+    },
+    {
```

Comment:
can you add test case with custom attribute not placed on first position?

Reply:
@armano2 added in 439906d, 663dd4f